.gitignore

```diff
@@ -23,3 +23,4 @@ imported_assets
 
 example_showcase_config.ron
 example-showcase-reports/
+/crates/bevy_ecs/.zed
```

Comment:
This seems pretty odd. I would at least expect it to cover files in all crates, and maybe place it with the other editor specific ignores (e.g. `/.idea` and `/.vscode` above)